toolchain/check/import_cpp.cpp

```diff
@@ -113,14 +116,19 @@ static auto AddNamespace(Context& context, PackageNameId cpp_package_id,
 }
 
 auto ImportCppFiles(Context& context, llvm::StringRef importing_file_path,
+                    std::unique_ptr<clang::ASTUnit>* ast,
```

Comment:
I think we'd pass-by-reference, not by pointer, since the `ast` pointer itself is not stored. That'd also eliminate the CHECK on 127.
```suggestion
                    std::unique_ptr<clang::ASTUnit>& ast,
```

Though, had you considered having `ImportCppFiles` return `std::unique_ptr<clang::ASTUnit>` instead of taking the return value as a parameter?

Also, other style note -- since this is an output parameter, it should typically be [the last parameter](https://google.github.io/styleguide/cppguide.html#Inputs_and_Outputs). Though that's moot if you just return the pointer.

Reply:
Done.